bin/node/runtime/src/lib.rs

```diff
@@ -497,6 +498,13 @@ parameter_types! {
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
 	pub const MaxNominatorRewardedPerValidator: u32 = 256;
 	pub OffchainRepeat: BlockNumber = 5;
+	pub const MaxNominations: u32 = MAX_NOMINATIONS;
+	pub const MaxUnappliedSlashes: u32 = 1_000;
+	pub const MaxNbOfInvulnerables: u32 = 10;
```

Comment:
not a fan of this `Nb`. If so, all of them should have it, i.e why not `MaxNbOfUnappliedSlashes`? (and I am not hinting at doing that, just drop all of them please.)

Reply:
Done